fe/src/main/java/org/apache/doris/external/elasticsearch/EsFieldInfo.java

```diff
@@ -0,0 +1,47 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.external.elasticsearch;
+
+import java.util.Map;
+
+/**
+ * EsFieldInfo
+ *
+ * @author stalary
+ * @since 2020/06/10
+ */
+public class EsFieldInfo {
+    
+    private Map<String, String> fetchFields;
+    
+    private Map<String, String> docValueFields;
+    
+    public EsFieldInfo(Map<String, String> fetchFields, Map<String, String> docValueFields) {
+        this.fetchFields = fetchFields;
+        this.docValueFields = docValueFields;
+    }
+    
+    public Map<String, String> getFetchFields() {
+        return fetchFields;
+    }
+    
+    public Map<String, String> getDocValueFields() {
+        return docValueFields;
+    }
+    
```

Comment:
Prefer add  static `parseXXX` or `fromXXXX` for this plain object to parse from response